8.0/debian/buster/developer/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM php:8.0-buster
+MAINTAINER "Bence Gazder"
+LABEL maintainer="bence.gazder@apicore.de"
+
+ENV COMPOSER_VERSION 2.0.11
+
+# install dependencies
+RUN mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini" && \ 
+    apt-get update && apt-get install -y \
+    supervisor \
+    cron \
+    ca-certificates \
+    libfreetype6 \
+    libfreetype6-dev \
+    libssl1.0 \
+    libpng16-16 \
+    libjpeg62-turbo \
+    libjpeg62-turbo-dev \
+    mcrypt \
+    libcurl4-openssl-dev \
+    libpng-dev \
+    libicu-dev \
+    libonig-dev \
+    zip \
+    libzip-dev \
+    $PHPIZE_DEPS \
+    nano \
+    git \
+    openssh-client \
+    sudo \
+# configure extensions
+    && docker-php-ext-configure intl --enable-intl \
+    && docker-php-ext-configure gd --with-freetype --with-jpeg \
+# install extensions
+    && docker-php-ext-install \
+    mbstring \
+    intl \
+    curl \
+    pdo_mysql \
+    exif \
+    zip \
+    gd \
+    bcmath \
+    opcache \
+    pcntl \
+    sockets \
+# pecl extensions
+    && pecl install xdebug \
+    && docker-php-ext-enable xdebug \
+# composer
+    && curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer --version=${COMPOSER_VERSION} \
+    && chmod +x /usr/local/bin/composer \
+# cleanup
+    && docker-php-source delete \
+    && apt-get purge -y \
+    libfreetype6-dev \
+    libjpeg62-turbo-dev \
+    libcurl4-openssl-dev \
+    libpng-dev \
+    libicu-dev \
+    libzip-dev \
+    $PHPIZE_DEPS \
+    && apt-get autoclean -y \
```

Comment:
`apt-get clean` does a better job. According to this it already implicitly runs in official debian images: https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105 So if the php guys did not mess with it, it shouldn't matter.

However, don't forget to `rm -rf /var/lib/apt/lists/*` as that frees up some space.

Also, these should be last commands in the apt-get chain, so put `apt-get autoremove -y` before them and not after.

The same applies to the other images as well.

Reply:
Will this be enough?
```
    && apt-get clean -y \
    && rm -rf /var/lib/apt/lists/* \
```

Or should I readd `apt-get autoremove -y`?

Reply:
Re-add `apt-get autoremove -y`, it should be there.

According to the debian docker docs `apt-get clean -y` should be optional, but leaving it here should also be okay, as a safety net in case something changes upstream.

Reply:
readded